bigtable-hbase-1.x-parent/bigtable-hbase-1.x-shaded/README.md

```diff
@@ -0,0 +1,21 @@
+This artifact is used in situations where dependencies don't converge properly.  Hadoop is one of
+those environments.  Hadoop uses really old versions of netty, protobuf and guava.  The simplest
```

Comment:
This isn't quite right. It implies that users should use the -shaded variant for hadoop, when they should be using the -hadoop variant. I think the description should be updated describe a general legacy application that uses both old versions of protobuf. And in a last sentence mention that hadoop is a special case that we handle via the -hadoop artifact